SimTracker/TrackTriggerAssociation/interface/TTStubAssociationMap.h

```diff
@@ -100,7 +120,7 @@ TTStubAssociationMap<T>::TTStubAssociationMap() {
   /// Set default data members
   stubToTrackingParticleMap.clear();
   trackingParticleToStubVectorMap.clear();
-  edm::RefProd<TTClusterAssociationMap<T> >* aRefProd = new edm::RefProd<TTClusterAssociationMap<T> >();
+  edm::RefProd<TTClusterAssociationMap<T>>* aRefProd = new edm::RefProd<TTClusterAssociationMap<T>>();
```

Comment:
potential memory leak

Reply:
Initialization of aRefProd unnecessary, so removed.